raindrops.md

```diff
@@ -6,10 +6,10 @@
 
 ## Examples
 
-- 28's prime-factorization is 2, 2, 7.
+- 28's factors are 2, 4, **7**, 14.
   - In raindrop-speak, this would be a simple "Plong".
-- 1755 prime-factorization is 3, 3, 3, 5, 13.
+- 30's factors are 2, **3**, **5**, 6, 15.
```

Comment:
Last thing to consider before merge, is the highlighted/bold numbers?  Should we remove them or explain why they are highlighted/bold?